napari/utils/colormaps/_tests/test_colormaps.py

```diff
@@ -36,11 +40,13 @@ def test_increment_unnamed_colormap():
         'perceptually_uniform',
         '[unnamed colormap 1]',
     ]
-    assert increment_unnamed_colormap(names) == '[unnamed colormap 2]'
+    assert increment_unnamed_colormap(names)[0] == '[unnamed colormap 2]'
```

Comment:
Does that needs a trans ? I know it's tests, but what if it runs on a user machine's with translations ?

Reply:
We can add ir, just in case, good catch

Reply:
Actually no need. The first item is always the name of the colormap, which is not translated.